scripts/compile-clean.js

```diff
@@ -10,10 +10,13 @@
  */
 
 const { rimraf } = require('rimraf');
+const { euiBuildTimeAliasTearDown } = require('./utils'); // Import the utility function
 
 rimraf.sync('.cache-loader');
 rimraf.sync('dist');
 rimraf.sync('lib');
 rimraf.sync('es');
 rimraf.sync('test-env');
 rimraf.sync('types');
+
+euiBuildTimeAliasTearDown(); // Call the utility function
```

Comment:
This should be surrounded by EUI build time alias markers to make the removal of EUI aliasing easier for us when we decide to do it

```suggestion
/* OUI -> EUI Aliases */
euiBuildTimeAliasTearDown();
/* End of Aliases */
```